lpa-submissions-e2e-tests/cypress/support/common/verifyPanelText.js

```diff
@@ -0,0 +1,9 @@
+import { panelBody } from '../../support/PageObjects/common-page-objects';
+
+module.exports = (email) => {
+  panelBody()
+    .invoke('text')
+    .then((text) => {
+      expect(text).to.contain(email);
```

Comment:
it should be expect(text).to.equal(email);

Reply:
I don't think so, there's extra text as well as the email.